src/Examples/Examples/Mapping/Types/SearchAsYouTypePage.cs

```diff
@@ -1,18 +1,23 @@
+using System.Collections.Generic;
 using Elastic.Xunit.XunitPlumbing;
 using Nest;
 
 namespace Examples.Mapping.Types
 {
 	public class SearchAsYouTypePage : ExampleBase
 	{
-		[U(Skip = "Example not implemented")]
+		[U]
 		public void Line18()
 		{
 			// tag::6f31f9cfe0dd741ccad4af62ba8f815e[]
-			var response0 = new SearchResponse<object>();
+			var createIndexResponse = client.Indices.Create("my_index", c => c
+				.Map(m => m
+					.Properties(p => p.SearchAsYouType(t => t.Name("my_field")))
```

Comment:
Could you put the `.SearchAsYouType(t => t` and `.Name("my_field"))` on separate, indented lines? This example will go into the Elasticsearch reference documentation, so it'll help for consistency